### PR TITLE
[SDPA-4708][SDPA-4714][SDPA-4716] card nav and card promo component

### PR DESCRIPTION
**Pull request by rubyllamadora**
**JIRA issue:** 
https://digital-engagement.atlassian.net/browse/SDPA-4708
https://digital-engagement.atlassian.net/browse/SDPA-4714
https://digital-engagement.atlassian.net/browse/SDPA-4716
https://digital-engagement.atlassian.net/browse/SDPA-4922
https://digital-engagement.atlassian.net/browse/SDPA-4546

### Changed

1.  Added Card Navigation V2 component and added config to connect to BE content
2. Updated Card Promo component and added config to connect BE content
3. Updated stories 

### Screenshots

https://storybook-ripple-pr-817.lagoon.vicsdp.amazee.io/?path=/story/molecules-card-card-navigation-v2--thumbnail

### CODE REVIEW

File: packages/components/Molecules/Card/CardNavigationV2.vue [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+      return formatted
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/scss/settings";
+@import "~@dpc-sdp/ripple-global/scss/tools";
+@import "scss/card";
```
Comment (tim-yao) [inline packages/components/Molecules/Card/CardNavigationV2.vue:88]: If there are any centralised style setting for all card types(e.g promo and nav), we can move to `scss/card.scss`.

File: packages/components/Molecules/Card/CardNavigationV2.vue [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+<script>
+import RplLink from '@dpc-sdp/ripple-link'
+import RplIcon from '@dpc-sdp/ripple-icon'
+import RplResponsiveImg from '@dpc-sdp/ripple-responsive-img'
+import formatdate from '@dpc-sdp/ripple-global/mixins/formatdate'
+import card from '@dpc-sdp/ripple-card/mixins/card'
+
+export default {
+  name: 'RplCardNavigationV2',
```
Comment (tim-yao) [inline packages/components/Molecules/Card/CardNavigationV2.vue:30]: Dylan didn't use V2 in name for his new promo, do you want to do the same?

**Review by tim-yao — APPROVED** [review]

File: packages/components/Molecules/Card/CardNavigationV2.vue (hunk 1 above)
Comment (rubyllamadora) [reply packages/components/Molecules/Card/CardNavigationV2.vue:88]: Good idea. I can see title, summary, date and tag/topic having the same styling. I have another PR to update Promo I'll do it in there.

File: packages/components/Molecules/Card/CardNavigationV2.vue (hunk 2 above)
Comment (rubyllamadora) [reply packages/components/Molecules/Card/CardNavigationV2.vue:30]: I actually prefer having no V2 (we don't need to rename as well when we remove unused cards). I'll update this to `RplCardNav` so it aligns with the shorten RplCardPromo (:

File: examples/vic-gov-au/nuxt.config.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ export default {
   * See https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-tide
   */
   tide: {
-    baseUrl: process.env.CONTENT_API_SERVER,
+    baseUrl: 'http://nginx-php-content-vic-pr-999.lagoon.vicsdp.amazee.io/',
```
Comment (rubyllamadora) [inline examples/vic-gov-au/nuxt.config.js:121]:
```suggestion
    baseUrl: process.env.CONTENT_API_SERVER,
```

**Comment by nicksantamaria** [comment]
@rubyllamadora the builds are deploying, but failing on the jira notifications. You can access this PR here - https://storybook.pr-817.ripple.sdp1.amazee.io

File: examples/vic-gov-au/nuxt.config.js [hunk 4]
```diff
@@ -128,7 +128,7 @@ export default {
   * See https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-tide
   */
   tide: {
-    baseUrl: process.env.CONTENT_API_SERVER,
+    baseUrl: 'http://nginx-php-content-vic-pr-1015.lagoon.vicsdp.amazee.io/',
```
Comment (rubyllamadora) [inline examples/vic-gov-au/nuxt.config.js:131]:
```suggestion
    baseUrl: process.env.CONTENT_API_SERVER,
```

**Pull request merged**